src/bridge/cockpitpackages.c

```diff
@@ -884,7 +884,7 @@ package_content (CockpitPackages *packages,
       g_free (chosen);
       chosen = NULL;
 
-      bytes = cockpit_web_response_negotiation (filename, package->paths, language, &chosen, &error);
+      bytes = cockpit_web_response_negotiation (filename, package ? package->paths : NULL, language, &chosen, &error);
```

Comment:
So if `package` would really be `NULL` here then `cockpit_packages_resolve()` would be broken. I wonder if it might not be nicer to add a `g_assert (package)` further up?